Sming/system/m_printf.cpp

```diff
@@ -11,39 +11,33 @@ Descr: embedded very simple version of printf with float support
 
 #define MPRINTF_BUF_SIZE 256
 
-#define OVERFLOW_GUARD 24
-
 static void defaultPrintChar(uart_t *uart, char c) {
-	return uart_tx_one_char(c);
+    return uart_tx_one_char(c);
```

Comment:
The are multiple white-space changes that are not related to this PR. Please, revert them.